### ISSUE TITLE
GT crash due to missing file?

### ISSUE THREAD

**Issue by Prometheus0000** [GTNewHorizons/GT-New-Horizons-Modpack#8050]
#### Which modpack version are you using?
Stripped 2.1.0.7
#
[fml-client-latest.log](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/files/6644446/fml-client-latest.log)
I got a weird error when crashing while testing stuff. There's no crashlog. The file is indeed not there in the jar file, but it loaded just fine before and after this.

**Comment by KaneHart** [comment]
Material Properties Config I think might be causing this issue.

I:MaterialID_609=609 ?

Just a guess.

**Comment by TimeConqueror** [comment]
Nope, lack of the texture can't produce such a crash. Maybe it's just a OutOfMemoryError and you have hs_err####.log instead.

**Comment by Prometheus0000** [comment]
I have those, but they're from months ago.
